src/components/results/guide/personality/question.js

```diff
@@ -7,9 +7,9 @@ import style from "./style.scss";
 
 function Question({question}) {
   const [showContent, setShowContent] = useState(false);
-  const [introduction, ...content] = question.text.split("\n\n\n");
-  const [number, ...remainingText] = content.join("\n\n\n").split(":");
-  const text = remainingText.join(":").trim();
+  const introduction = question.introduction
```

Comment:
We might want to fall back to the old way or invalidate the old cache, I think the guide could render from the cache and be missing the new fields